tesseract_geometry/include/tesseract_geometry/impl/polygon_mesh.h

```diff
@@ -201,7 +203,7 @@ class PolygonMesh : public Geometry
    *
    * @return The vertex normal vector
    */
-  std::shared_ptr<const tesseract_common::VectorVector3d> getNormals() const { return normals_; }
+  const std::shared_ptr<const tesseract_common::VectorVector3d>& getNormals() const { return normals_; }
```

Comment:
Just curious, why the switch to const&?

Reply:
SWIG wasn't detecting the type properly without const&